### DISCUSSION TITLE
SmallRye fault tolerance version 5.3.1 from bom is not being taken

### DISCUSSION THREAD

**Discussion by matiasgonsal** [quarkusio/quarkus#23981] [Q&A]
I'm trying to upgrade the SmallRye fault tolerance dependency to the latest version 5.3.1. Even if a create a new quarkus application the SmallRye fault tolerance version is still 5.2.1

mvn io.quarkus.platform:quarkus-maven-plugin:2.7.2.Final:create \
    -DprojectGroupId=org.acme \
    -DprojectArtifactId=getting-started \
    -DclassName="org.acme.getting.started.GreetingResource" \
    -Dpath="/hello" \
    -DbuildTool=gradle
    
    importing: implementation 'io.quarkus:quarkus-smallrye-fault-tolerance' should bring up the version 5.3.1 as per the quarkus latest bom: https://github.com/quarkusio/quarkus/blob/main/bom/application/pom.xml#L48

[a comment by an automated account is not shown]

**Comment by gsmet** [comment]
The 2.7 branch is using 5.2.1.Final. That's why you have this version.

**Reply by matiasgonsal** [reply]
Thank you Guillaume!
Is there any Quarkus release planned to include small-rye fault-tolerance 5.3.1?
Thanks.

**Reply by gsmet** [reply]
Yes, 2.8 but it's not for now.

Are you waiting for a particular fix?

**Reply by matiasgonsal** [reply]
Yes, I'm waiting for this fix: https://github.com/quarkusio/quarkus/issues/22832
I will give it a try once release 2.8 is available!
Thank you.

**Reply by gsmet** [reply]
OK. Schedule for 2.8 is here: https://github.com/quarkusio/quarkus/wiki/Release-Planning .

We can't really backport the upgrade to SmallRye FT 5.3 as it's a feature release.
